api.json

```
@@ -268,6 +349,27 @@
           }
         }
       },
+      "GetPaymentListResponse": {
+        "type": "object",
+        "required": ["data", "status", "message"],
+        "properties": {
+          "data": {
+            "type": "array",
+            "x-go-custom-tag": "bson:\"data\"",
+            "items": {
+              "$ref": "#/components/paymentByMemberId"
```

Ini kalau nama componentnya jadi `Payment` aja gimana ya ?

```suggestion
              "$ref": "#/components/Payment"
```